### PR TITLE
docs(datadog_agent source): Brush off trace support docs

### PR DESCRIPTION
**Pull request by jszwedko**
Signed-off-by: Jesse Szwedko <jesse.szwedko@datadoghq.com>

### CODE REVIEW

File: website/cue/reference/components/sources/datadog_agent.cue [hunk 1]
```diff
@@ -232,14 +232,23 @@ components: sources: datadog_agent: {
 					metrics.url: http://"<VECTOR_HOST>:<SOURCE_PORT>" # Use https if SSL is enabled in Vector source configuration
 				```
 
+				In order to send traces the [Datadog Agent](\(urls.datadog_agent_doc)) configuration must be updated with the
+				following options:
+
+				```yaml
+				apm_config:
+					apm_dd_url: http://"<VECTOR_HOST>:<SOURCE_PORT>" # Use https if SSL is enabled in Vector source configuration
```
Comment (spencergilbert) [inline website/cue/reference/components/sources/datadog_agent.cue:240]: Isn't there a `vector.traces.url` now?

Comment (jszwedko) [reply website/cue/reference/components/sources/datadog_agent.cue:240]: I couldn't find it but I'll take another look if you think it exists 😄

Comment (spencergilbert) [reply website/cue/reference/components/sources/datadog_agent.cue:240]: I _think_ it became a thing in a fairly recent agent version. Let me take a look too.

Comment (spencergilbert) [reply website/cue/reference/components/sources/datadog_agent.cue:240]: https://cs.github.com/DataDog/datadog-agent/blob/a728670757b568e7cb4e1deb7de6fad0b80cbdd2/releasenotes/notes/vector-traces-override-bebe167f286f7a3d.yaml?q=vector

**Review by spencergilbert — APPROVED** [review]
It may be worth noting when support for each of these were added to the Agent, and include the old style of config as an alternative? [Example](https://github.com/vectordotdev/helm-charts/blob/develop/charts/vector/docs/quickstart.md)

**Pull request merged**